src/System.Text.Json/tests/Serialization/Stream.ReadTests.cs

```diff
@@ -79,5 +81,17 @@ public static async Task ReadPrimitivesWithTrailingTriviaAsync()
                 Assert.Equal(1, i);
             }
         }
+
+        [Fact]
+        public static async Task ReadCollectionPassingNullValue()
+        {
+            using (MemoryStream stream = new MemoryStream(Encoding.UTF8.GetBytes("null")))
+            {
+                JsonSerializerOptions options = new JsonSerializerOptions();
```

Comment:
nit: This is not necessary since the default options will be used anyway if you don't pass anything to the `JsonSerializer`:
https://github.com/dotnet/corefx/blob/23faf5d032feef556f5399b46a92a6e4aef7d84a/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs#L19

So, just passing the stream (without options) should be sufficient.